### PR TITLE
PerformanceDisplay styled by CSS instead of JS.

### PR DESCRIPTION
**Pull request by emackey**
Moved some JS to CSS.  You can turn this display on from the URL bar with `?stats=true`.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+### 1.11 - 2015-07-01
+
+* Breaking changes
+  * PerformanceDisplay (private API) no longer takes style options.  It is now styled with app-overridable CSS.  Use `scene.debugShowFramesPerSecond` to enable it as before.
```
Comment (mramato) [inline CHANGES.md]: I would change this to something like "The performance statistics displayed by setting `scene.debugShowFramesPerSecond` to `true` can now be styled using the `cesium-performanceDisplay` CSS classes in `shared.css`.

File: Source/Widgets/shared.css [hunk 2]
```diff
@@ -80,4 +80,26 @@
...
+    top: 50px;
+    right: 10px;
+}
+
+.cesium-performanceDisplay {
+    background-color: rgba(40, 40, 40, 0.7);
+    padding: 7px;
+    border-radius: 5px;
+    border: 1px solid #444;
+    font: 12px sans-serif;
```
Comment (hpinkos) [inline Source/Widgets/shared.css]: This should be bold.

**Comment by hpinkos** [comment]
@emackey everything else looks great! thanks!

**Comment by emackey** [comment]
Thanks both.  Ready.

**Comment by hpinkos** [comment]
Great! thanks @emackey

**Pull request merged**